### DISCUSSION TITLE
Is it possible to keep the snippet in its last state?

### DISCUSSION THREAD

**Discussion by YiiGuxing** [massCodeIO/massCode#143] [Q&A] [answered]
Is it possible to keep the snippet in its last state?
For example:
- Preview Code state
- Preview Markdown state

**Comment by antonreshetov** [comment, marked as answer]
Currently no.

**Reply by YiiGuxing** [reply]
Can you add this functionality? Each snippet keeps its own last state, it's really useful, hope you think about it, thanks.

**Reply by antonreshetov** [reply]
@YiiGuxing I took notes, maybe in the future I will implement it.

**Reply by YiiGuxing** [reply]
Thanks!
